docs/framework/data/adonet/sql/linq/programming-guide.md

```diff
@@ -34,5 +34,5 @@ This section contains information about how to create and use your [!INCLUDE[vbt
  [Stored Procedures](../../../../../../docs/framework/data/adonet/sql/linq/stored-procedures.md)  
  Includes links to topics that illustrate how to use stored procedures.  
   
- [Introduction to LINQ](https://msdn.microsoft.com/library/24dddf19-12a0-4707-a4bc-eba4fa7f219e)  
+ [Introduction to LINQ](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/bb397897(v=vs.120))  
```

Comment:
Split these in two:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/introduction-to-linq
https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/concepts/linq/introduction-to-linq